diff --git a/src/components/App.js b/src/components/App.js
index c39c931..16c3bb5 100644
--- a/src/components/App.js
+++ b/src/components/App.js
@@ -4,7 +4,6 @@ import { Link } from 'react-router-dom';
 import { getIsUsingFallback } from '@/adapters/web3/init';
 
 import Error from './Error';
-import Logo from '@/svgs/Logo';
 import Search from './Search';
 import Switcher from './ReactRouter/Switcher';
 
@@ -16,7 +15,7 @@ const App = () => (
           <h1 className='mix-header__title'>Block Explorer</h1>
         </Link>
         <p className='mix-header__fineprint'>Connected to {getIsUsingFallback()
-          ? 'MIX (rpc2.mix-blockchain.org)'
+          ? 'Pantheon Ethereum private Docker network (rpcHostPlaceHolder)'
           : 'your Web3 browser extension (e.g. Metamask)'}
         </p>
       </div>
@@ -24,13 +23,6 @@ const App = () => (
     </header>
     <Switcher />
     <Error />
-    <a href='https://www.mix-blockchain.org/'>
-      <footer className='mix-footer'>
-        <p>powered by</p>
-        <Logo className='mix-footer__logo' />
-        <p>MIX Blockchain</p>
-      </footer>
-    </a>
   </div>
 );
 
